### PR TITLE
Add image, change login wording following the style guide

### PR DESCRIPTION
**Pull request by HLeithner**
Add Icon

### CODE REVIEW

File: administrator/language/en-GB/plg_system_webauthn.ini [hunk 1]
```diff
@@ -35,7 +35,7 @@ PLG_SYSTEM_WEBAUTHN_HEADER="Passkey Login"
 PLG_SYSTEM_WEBAUTHN_LBL_DEFAULT_AUTHENTICATOR="Generic Passkey"
 PLG_SYSTEM_WEBAUTHN_LBL_DEFAULT_AUTHENTICATOR_LABEL="%s added on %s"
 PLG_SYSTEM_WEBAUTHN_LOGIN_DESC="Login without a password using the browser passkey standard in compatible browsers. You need to have already set up passkey authentication in your user profile."
-PLG_SYSTEM_WEBAUTHN_LOGIN_LABEL="Passkey"
+PLG_SYSTEM_WEBAUTHN_LOGIN_LABEL="Sign in with a passkey"
```
Comment (brianteeman) [inline administrator/language/en-GB/plg_system_webauthn.ini:38]: not a fan of this change - its overly verbose and I assume you only did it to make the word passkey lowercase which is not required in this case

Comment (HLeithner) [reply administrator/language/en-GB/plg_system_webauthn.ini:38]: Its requires by the style guide, the reason is awareness and I think thats the Main reason for the change to passkeys

Comment (brianteeman) [reply administrator/language/en-GB/plg_system_webauthn.ini:38]: please posint to the part of the styleguide that requires this - I didnt see it

Comment (brianteeman) [reply administrator/language/en-GB/plg_system_webauthn.ini:38]: sorry - i saw it now

**Comment by brianteeman** [comment]
if you can fix the mising image then I will merge it

**Comment by HLeithner** [comment]
If I get the time and the skill ;-) to create that image I will do it but could take some time.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**